core-bundle/src/Resources/contao/classes/Crawl.php

```diff
@@ -285,9 +282,26 @@ private function createLogger(Factory $factory, array $activeSubscribers, string
 		return $logger;
 	}
 
+	private function getLogDir(): string
+	{
+		if (null !== $this->logDir)
+		{
+			return $this->logDir;
+		}
+
+		$this->logDir = sprintf('%s/%s/contao-crawl', sys_get_temp_dir(), md5(System::getContainer()->getParameter('kernel.project_dir')));
```

Comment:
Works fine. I would prefer the use of rtrim to avoid double slashes on some systems.

```suggestion
		$this->logDir = sprintf('%s/%s/contao-crawl', rtrim(sys_get_temp_dir(), '/'), md5(System::getContainer()->getParameter('kernel.project_dir')));
```

Reply:
Double slashes in paths are not a problem on any platform IMHO. Also, there are several places in Symfony where `sys_get_temp_dir()` is used and none of them uses `rtrim()`, so it does not make sense to add this only here.

Reply:
Ok, I was not sure if this is valid. On Strato I get double slashes and it works.